dev-libs/libprelude/libprelude-3.1.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5}} )
+
+USE_RUBY="ruby21"
+
+inherit autotools eutils python-r1 ruby-single
+
+DESCRIPTION="Prelude-SIEM Framework Library"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc lua python ruby perl"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="dev-libs/libgcrypt:0=
+	net-libs/gnutls
+	perl? ( dev-lang/perl virtual/perl-ExtUtils-MakeMaker )
+	lua? ( dev-lang/lua:* )
+	ruby? ( ${RUBY_DEPS} )
+	python? ( ${PYTHON_DEPS} )"
+
+DEPEND="${RDEPEND}
+	doc? ( dev-util/gtk-doc )
+	sys-devel/flex
+	virtual/yacc
+	>=dev-lang/swig-3.0.7
```

Comment:
We provide swig files if the system does not provide it. if not, swig >= 3.0.7 is required, this range do not change. 3.0.7 fix an issue with python3.5